src/modules/app/StateType.js

```diff
@@ -10,7 +10,7 @@ import {
   LanguageModel
 } from '@integreat-app/integreat-api-client'
 import Moment from 'moment'
-import { DEFAULT_LANGUAGE } from '../i18n/components/I18nProvider'
```

Comment:
I would prefere to stick to the i18n name. It is the default for basically every software project.

Also we should try not to rename files if not really needed. (Because of merge conflicts etc.)